libs/wire-api/src/Wire/API/Error/Brig.hs

```diff
@@ -55,6 +55,12 @@ data BrigError
   | MLSProtocolError
   | MLSDuplicatePublicKey
   | InvalidPhone
+  | PasswordExists
+  | AccountSuspended
+  | AccountEphemeral
+  | AccountPending
+  | LoginCodeAuthenticationFailed
```

Comment:
Redundant with CodeAuthenticationFailed and CodeAuthenticationRequired :)

Reply:
Thanks for catching this. Fixed.